tools/ci_build/github/azure-pipelines/linux-ort-pytorch-ci-pipeline.yml

```diff
@@ -1,16 +1,29 @@
+trigger: none
+
 jobs:
-- job: Linux_CI_Dev
+- job: Onnxruntime_Linux_pytorch_export
+
   timeoutInMinutes: 60
-  workspace:
-    clean: all
-  pool: Linux-CPU
+
   steps:
   - checkout: self
     clean: true
     submodules: recursive
 
   - script: >
-      PYTHONPATH='$(Build.SourcesDirectory)/tools' pytest -v python tools/python/test_custom_ops_pytorch_exporter.py
-    displayName: 'Test Custom Ops PyTorch Export'
+      tools/ci_build/github/linux/run_dockerbuild.sh
```

Comment:
I dont remember what is inside `tools/ci_build/github/linux/run_dockerbuild.sh`, but I think it only builds ort and creates the wheel package. You may have to do a `pip install $(Build.BinariesDirectory)/dist/*.whl` or something like that

Try running this script on your machine and then do a local `docker run` to test whether ort is installed